projects/packages/forms/src/contact-form/css/grunion.css

```diff
@@ -386,27 +386,43 @@
 
 .contact-form .contact-form__select-wrapper {
 	position: relative;
+
+	/*
+	 * Remove padding set on .grunion-field. We want the nested input to have no space around it.
+	 * Instead the padding is added to the `select` element itself.
+	 */
+	 padding: 0;
+}
+
+.contact-form .contact-form__select-wrapper .jetpack-field-dropdown__icon {
+	position: relative;
 }
 
-.contact-form .contact-form__select-wrapper::after {
+.contact-form .contact-form__select-wrapper .jetpack-field-dropdown__icon::after {
 	position: absolute;
-	inset-inline-end: 20px;
-	top: 50%;
+    inset-inline-end: 20px;
+    top: 50%;
+    content: "";
+    display: block;
+    width: calc(1em / 2.5);
+    height: calc(1em / 2.5);
+    border-bottom: calc(1em / 10) solid;
+    border-right: calc(1em / 10) solid;
```

Comment:
Tiny formatting nit: for some reason the spacing here (and further below) has switched to spaces. Not a big deal but it's affecting how the diff looks in Github.